app/src/assets/scss/custom/ui/molecules/product-category-banner/product-category-banner.scss

```diff
@@ -0,0 +1,9 @@
+.product-wrapper-bg-pink {
+    background: url("../../../../../img/grocery-bg.png") no-repeat center center;
+    background-size: cover;
+}
+
+.product-wrapper-bg-green {
+    background: url("../../../../../img/grocery-bg-1.png") no-repeat center center;
+    background-size: cover;
+}
```

Comment:
takie klasy możesz robić w pliku scss bardziej common czyli np backgrounds.scss no chyba że już nigdzie indziej nie będą uzywane niż tylko w tym componencie (ale na moje oko lepiej wrzucic w oddzielny plik)

Reply:
your fault, everything is ok